### ISSUE TITLE
[Feature Request] TypeScript support

### ISSUE THREAD

**Issue by stargazing-dino** [romainberger/react-portal-tooltip#92]
Would love to see a types file defined for this library as it really is good.

**Comment by Redmega** [comment]
PRs always welcome :wink:

I've been using a lot of Typescript lately. If you don't care to make a PR adding typings (or converting to TS, though I'm not sure how @romainberger feels about a full conversion) I might be able to get around to it eventually. I've been insanely busy so I haven't had the chance or motivation to check on this library in a while.

**Comment by romainberger** [comment]
Yeah I haven't used Typescript for now so if there is a possibility to keep it as is and add types separately that'd be cool 👍

**Comment by stargazing-dino** [comment]
I've never written one but I think I could figure it out. If anyone else feels the urge to write one and get a PR in quick, please don't let me stop you

**Comment by naortor** [comment]
Hi, I created types @ definitely typed.

Now in PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45221
